less/notificationBar.less

```diff
@@ -7,7 +7,19 @@
 
 .notificationBar {
   display: inline-block;
+  position: relative;
+  z-index: 1000;
```

Comment:
@luixxiul That appears to cause the type of problems we faced in the design discussion last night. Namely, interacting with some of the document elements will cause the shadow to vanish; often only regions of it will vanish.

![odd-shadow](https://cloud.githubusercontent.com/assets/815158/22484179/d52a8ad0-e7c5-11e6-9cf7-105d9107367d.gif)